### DISCUSSION TITLE
LDAP LOGIN ERROR (c2):  select "users".* from "users" where "providerId" = $1 and "providerKey" = $2 - illegal Byte-Sequence for »UTF8«: 0x00

### DISCUSSION THREAD

**Discussion by richardklingler** [requarks/wiki#4409] [Help / Questions]
Good afternoon (o;

Just did a fresh install of wikijs on a Debian 10 system and wanted to use our internal LDAP/AD for authentication...

But as soon someone tries to log in I get this error in the console:

2021-09-12T09:39:22.266Z [MASTER] warn: LDAP LOGIN ERROR (c2):  select "users".* from "users" where "providerId" = $1 and "providerKey" = $2 - illegal Byte-Sequence for »UTF8«: 0x00

LDAP authentication works as it otherwise prints unknown username/password....

But form the so many unanswered questions here I believe i need to use some other wiki system (o;

**Comment by millercmary** [comment]
I'm having LDAP issues too, and equally concerned about few replies. Did you choose another wiki? If so, which?
